Changelog.rst

```diff
@@ -19,6 +19,10 @@ an overview of what's new in Celery 5.5.
 What's Changed
 ~~~~~~~~~~~~~~
 
+- Add a new configuration setting `worker_eta_task_limit` to limit the number of ETA/countdown tasks 
```

Comment:
please remove this from changelog. this has to be in configuration and relevant area